products/red-hat-jboss-eap.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /jboss-eap
+-   /jboss
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes
+changelogTemplate: "https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/{{'__LATEST__'|split:'.'|pop|join:'.'}}"
+releaseDateColumn: true
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+extendedSupportColumn: <abbr title="Extended Life Support">ELS</abbr> 1
```

Comment:
It would be better to use the full name here.

Reply:
Done